src/components/footer/Footer.tsx

```diff
@@ -0,0 +1,21 @@
+import {Text, View} from 'react-native';
+
+import {Button} from '@components/button';
+import styles from './styles';
+
+const Footer = ({
+  text,
+  buttonLabel,
+  onPressLogin,
+}: AlreadySignedUpProps) => {
```

Comment:
`AlreadySignedUpProps`??

I assume this footer is used in both login and signup, right?

also name of the component is not correct, it should signify its being used on Auth screens, its not like this footer will be used everywhere in the app